query-node/mappings/src/bounty.ts

```diff
@@ -0,0 +1,496 @@
+import { DatabaseManager, EventContext, StoreContext, SubstrateEvent } from '@joystream/hydra-common'
+import { BountyMetadata } from '@joystream/metadata-protobuf'
+import { AssuranceContractType, BountyActor, BountyId, EntryId, FundingType } from '@joystream/types/augment'
+import { MemberId } from '@joystream/types/common'
+import { BN } from '@polkadot/util'
+import {
+  Bounty,
+  BountyCanceledEvent,
+  BountyContribution,
+  BountyCreatedEvent,
+  BountyCreatorCherryWithdrawalEvent,
+  BountyEntrantWhitelist,
+  BountyEntry,
+  BountyEntryStatusCashedOut,
+  BountyEntryStatusPassed,
+  BountyEntryStatusRejected,
+  BountyEntryStatusWinner,
+  BountyEntryStatusWithdrawn,
+  BountyEntryStatusWorking,
+  BountyFundedEvent,
+  BountyFundingLimited,
+  BountyFundingPerpetual,
+  BountyFundingWithdrawalEvent,
+  BountyMaxFundingReachedEvent,
+  BountyRemovedEvent,
+  BountyStage,
+  BountyVetoedEvent,
+  ForumThread,
+  Membership,
+  OracleJudgmentSubmittedEvent,
+  WorkEntrantFundsWithdrawnEvent,
+  WorkEntryAnnouncedEvent,
+  WorkEntrySlashedEvent,
+  WorkEntryWithdrawnEvent,
+  WorkSubmittedEvent,
+} from 'query-node/dist/model'
+import { Bounty as BountyEvents } from '../generated/types'
+import { asBN, asInt32, deserializeMetadata, genericEventFields, perpareString } from './common'
+import { scheduleAtBlock } from './scheduler'
+
+/**
+ * Common helpers
+ */
+
+async function getBounty(store: DatabaseManager, bountyId: BountyId | string, relations?: string[]): Promise<Bounty> {
+  const bounty = await store.get(Bounty, { where: { id: bountyId }, relations })
+  if (!bounty) {
+    throw new Error(`Bounty not found by id: ${bountyId}`)
+  }
+  return bounty
+}
+
+async function getContribution(
+  store: DatabaseManager,
+  bountyId: BountyId,
+  contributor: string | undefined
+): Promise<BountyContribution> {
+  const contribution = await store.get(BountyContribution, { where: { bountyId, contributor } })
```

Comment:
The query always returns empty result, it looks like you need to modify the query slightly in order for it to work:
```suggestion
  const contribution = await store.get(BountyContribution, {
    where: { bounty: { id: bountyId.toString() }, contributor: { id: contributor } },
  })
```